suites/quincy/rados/tier-3_rados_test-4-node-ecpools.yaml

```diff
@@ -0,0 +1,252 @@
+# Suite is to be used to verify the EC 2+2 config on 4 nodes
+
+tests:
+  - test:
+      name: setup install pre-requisistes
+      desc: Setup phase to deploy the required pre-requisites for running the tests.
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: cluster deployment
+      desc: Execute the cluster deployment workflow.
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              base_cmd_args:
+                verbose: true
+              args:
+                # using 6.1 rc candidate
+                custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:6-177"
+                custom_repo: "http://download-node-02.eng.bos.redhat.com/rhel-9/rel-eng/RHCEPH-6/RHCEPH-6.1-RHEL-9-20230602.0/compose/Tools/x86_64/os/"
```

Comment:
I could not find any upgrade path in this test suite, any reason as to why we are hard coding the build here?

Reply:
Upgrade is in the test workflow. 

Need to check for inactive PGs and data unavailability scenario during upgrade,a nd since our regular workflow does not do that, have added upgrade workflow in the test case.